x-pack/plugins/observability_solution/logs_data_access/server/services/register_services.ts

```diff
@@ -5,16 +5,14 @@
  * 2.0.
  */
 
-import { Logger } from '@kbn/logging';
 import { createGetLogsRatesService } from './get_logs_rates_service';
+import { createGetLogsRateTimeseries } from './get_logs_rate_timeseries/get_logs_rate_timeseries';
+import { createGetLogErrorRateTimeseries } from './get_logs_error_rate_timeseries/get_logs_error_rate_timeseries';
 
-export interface RegisterServicesParams {
-  logger: Logger;
-  deps: {};
-}
-
-export function registerServices(params: RegisterServicesParams) {
+export function registerServices() {
```

Comment:
https://buildkite.com/elastic/kibana-pull-request/builds/219073#0190734a-b48b-4c2e-923d-f08d21c6b1f7
I guess the params are expected because of the usage in `x-pack/plugins/observability_solution/logs_data_access/server/plugin.ts` - at least the logger is passed there 🤔

Reply:
Probably yes, eslint also complained. Let me fix it

Reply:
https://github.com/elastic/kibana/pull/187221/commits/32c5121e7afe98e0696fd196cdeabdc8c091444d